tests/testthat/error/test-conditions.txt

```diff
@@ -111,3 +111,19 @@ ordered cast failures mention conversion
 > vec_cast(ordered("x"), ordered("y"))
 Error: Can't convert <ordered<5a425>> to <ordered<df698>>.
 
+
+incompatible size errors
+========================
+
+> stop_incompatible_size(1:2, 3:5, 2L, 3L, x_arg = "", y_arg = "")
+Error: Can't recycle size 2 to match size 3.
+
+> stop_incompatible_size(1:2, 3:5, 2L, 3L, x_arg = quote(foo), y_arg = "")
+Error: Can't recycle `foo` (size 2) to match size 3.
+
+> stop_incompatible_size(1:2, 3:5, 2L, 3L, x_arg = "", y_arg = "bar")
+Error: Can't recycle size 2 to match `bar` (size 3).
```

Comment:
I think match should only be used in this case — if we don't have the name of RHS, it should just be "to size 3"